### DISCUSSION TITLE
What's purpose of initializing bias like this?

### DISCUSSION THREAD

**Discussion by lixit** [Hon-Wong/Elysium#8] [Q&A] [answered]
I know this is for spatial pooling(selecting the most import patch)

But what is the purpose of setting to 0.02 every 5 to the bias weights?

https://github.com/Hon-Wong/Elysium/blob/5e6d14ed6939cde3cbffaba5424d5d929c38e492/models/rantselector.py#L34

**Comment by lixit** [comment, marked as answer]
If my understanding is right, it will encourage the selected patches more evenly distributed.

It's saying, pay a little more attention to the patches which spatial distributed evenly. (specifically, 108 out of 576)

**Reply by Hon-Wong** [reply]
Yes, you are right.
